site/remix.config.js

```diff
@@ -11,29 +11,29 @@ module.exports = {
   devServerPort: 8002,
   ignoredRouteFiles: ['.*'],
   mdx: async () => {
-    const [rehypeSlug] = await Promise.all([
-      import('rehype-slug').then((mod) => mod.default),
-    ]);
-
-    console.log(rehypeSlug);
+    const rehypeSlug = await import('rehype-slug');
+    const remarkPrism = await import('remark-prism');
+    const rehypeCloudinary = await import('rehype-local-image-to-cloudinary');
 
     return {
       remarkPlugins: [
         [
-          require('remark-prism'),
+          remarkPrism.default,
```

Comment:
nice, thank you! I tried a few combinations here and eventually gave up 😂

Reply:
the wonderful life of using esm in cjs :sweat\_smile: